test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/PravegaControllerK8sService.java

```diff
@@ -92,12 +92,12 @@ public boolean isRunning() {
                         .thenCompose(o -> {
                            Map<String, Object> spec = (Map<String, Object>) (((Map<String, Object>) o).get("spec"));
                            Map<String, Object> pravegaSpec = (Map<String, Object>) spec.get("pravega");
-                           Map<String, Object> bookkeeperSpec = (Map<String, Object>) spec.get("bookkeeper");
+                           //Map<String, Object> bookkeeperSpec = (Map<String, Object>) spec.get("bookkeeper");
 
                            int currentControllerCount = ((Double) pravegaSpec.get("controllerReplicas")).intValue();
                            int currentSegmentStoreCount = ((Double) pravegaSpec.get("segmentStoreReplicas")).intValue();
-                           int currentBookkeeperCount = ((Double) bookkeeperSpec.get("replicas")).intValue();
-                           log.debug("Current instance counts : Bookkeeper {} Controller {} SegmentStore {}.", currentBookkeeperCount,
+                           //int currentBookkeeperCount = ((Double) bookkeeperSpec.get("replicas")).intValue();
```

Comment:
Same here.

Reply:
Yes same issue as mentioned in above comment,
Will remove the commented line

Reply:
This changes not related to PR so revert the changes